apps/studio/components/interfaces/Settings/Logs/Logs.utils.ts

```diff
@@ -296,6 +296,24 @@ const calcChartStart = (params: Partial<LogsEndpointParams>): [Dayjs, string] =>
   return [its.add(-extendValue, trunc), trunc]
 }
 
+function getErrorCondition(table: LogsTableName): string {
+  switch (table) {
+    case 'edge_logs':
+      return 'response.status_code >= 400'
+    case 'postgres_logs':
+      return "parsed.error_severity IN ('ERROR', 'FATAL', 'PANIC')"
+    case 'function_logs':
+      return "metadata.level IN ('error', 'fatal')"
+    case 'auth_logs':
+      return "metadata.level = 'error' OR metadata.status >= 400"
+    case 'function_edge_logs':
+      return 'response.status_code >= 400'
```

Comment:
gotta review this conditions, can be ignored for now, only tested in edge_logs

Reply:
Looks fine, best to verify on a staging project